app/views/customers/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Customer: <%= @customer.name %></h1>
+
+<section>
+  <p>List of items:</p>
+  <ol>
+    <% @customer.items.each do |item| %>
+      <li><%= item.name %>: $<%= item.price %>, found at: <%= item.supermarket.name %></li>
+    <% end %>
+  </ol>
+  <%= form_with url: "/customers/#{@customer.id}/items", method: :post, data: { turbo: false } do |form| %> <!-- is this URL RESTful? -->
```

Comment:
This is exactly how we would want this url to be. That way we are establishing the connection between customers and items.